cocos/core/math/utils.ts

```diff
@@ -27,9 +27,17 @@ import * as bits from './bits';
 import { ValueType } from '../value-types';
 import { IVec3Like } from './type-define';
 
-const _d2r = Math.PI / 180.0;
+/**
+ * @en One angle in radian, equals to PI / 180
+ * @zh 弧度制中的单位角度，等同于 PI / 180
+ */
+export const RAD = Math.PI / 180.0;
 
-const _r2d = 180.0 / Math.PI;
+/**
+ * @en One radian in angle, equals to 180 / PI
+ * @zh 角度制中的单位弧度，等同于 180 / PI
+ */
+export const DEG = 180.0 / Math.PI;
```

Comment:
May be RAD_2_DEG is a better name.

Reply:
no, this is not a transform tool or method, it's a unit name instead.

Reply:
maybe DEG_PER_RAD ?

Reply:
updated https://github.com/cocos/cocos-engine/pull/16365/commits/4ea2b6ecdcc2a51490a359a8e14bbdbe903a858d